src/component/admin/users/UsersList/UserTypeCell/UserTypeCell.tsx

```diff
@@ -0,0 +1,29 @@
+import { MonetizationOn } from '@mui/icons-material';
+import { styled, Tooltip } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { TextCell } from 'component/common/Table/cells/TextCell/TextCell';
+
+interface IUserTypeCellProps {
+    value: boolean;
+}
+
+export const UserTypeCell = ({ value }: IUserTypeCellProps) => {
+    return (
+        <TextCell>
+            <ConditionallyRender
+                condition={value}
+                show={
+                    <Tooltip title="Paid user" arrow>
+                        <StyledMonetizationOn />
+                    </Tooltip>
+                }
+                elseShow="Free"
+            />
+        </TextCell>
+    );
+};
+
+const StyledMonetizationOn = styled(MonetizationOn)(({ theme }) => ({
```

Comment:
I prefer to have `styled` higher in the document structure, because it's easier to "scan" what the returned element from the very file is.

Reply:
Thoughts @olav @FredrikOseberg ? Also discussing on Slack, so I'll try to adapt if needed.